book/src/user/mining.md

```diff
@@ -0,0 +1,167 @@
+        "hash": "2e9296d48f036112541b39522b412c06057b2d55272933a5aff22e17aa1228cd",
+        "authdigest": "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+        "depends": [],
+        "fee": 1367,
+        "sigops": 35,
+        "required": false
+      }
+    ],
+    "coinbasetxn": {
+      "data": "0400008085202f89010000000000000000000000000000000000000000000000000000000000000000ffffffff050378651e00ffffffff04b4e4e60e0000000017a9140579e6348f398c5e78611da902ca457885cda2398738c94d010000000017a9145d190948e5a6982893512c6d269ea14e96018f7e8740787d010000000017a914931fec54c1fea86e574462cc32013f5400b8912987286bee000000000017a914d45cb1adffb5215a42720532a076f02c7c778c90870000000078651e000000000000000000000000",
+      "hash": "f77c29f032f4abe579faa891c8456602f848f423021db1f39578536742e8ff3e",
+      "authdigest": "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+      "depends": [],
+      "fee": -12852,
+      "sigops": 0,
+      "required": true
+    },
+    "longpollid": "0001992055c6e3ad7916770099070000000004516b4994",
+    "target": "0000000001a11f00000000000000000000000000000000000000000000000000",
+    "mintime": 1677004508,
+    "mutable": [
+      "time",
+      "transactions",
+      "prevblock"
+    ],
+    "noncerange": "00000000ffffffff",
+    "sigoplimit": 20000,
+    "sizelimit": 2000000,
+    "curtime": 1677004885,
+    "bits": "1c01a11f",
+    "height": 1992056,
+    "maxtime": 1677009907
+  },
+  "id": "curltest"
+}
+$ 
+```
+
+## Run a mining pool
+[#run-a-mining-pool]: #run-a-mining-pool
+
+Just point your mining pool software to the Zebra RPC endpoint (127.0.0.1:8232). Zebra supports all the RPC methods needed to run the [s-nomp](https://github.com/ZcashFoundation/s-nomp) mining pool software.
```

Comment:
I'm not sure if we should suggest that `s-nomp` is useful mining software?
It hasn't been updated for NU5, so I don't want to lead people towards it.

```suggestion
Just point your mining pool software to the Zebra RPC endpoint (127.0.0.1:8232). Zebra supports the RPC methods needed to run most mining pool software.
```